clientd3d/d3drender_world.c

```diff
@@ -147,6 +141,9 @@ long D3DRenderWorld(
 	IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_CULLMODE, D3DCULL_NONE);
 	SetZBias(gpD3DDevice, 1);
 
+	// Disable alpha testing and blending
```

Comment:
Is this comment correct?  It looks like it only disables alpha testing, not blending.